{"name": "Ошибка авторизации при неверных параметрах", "status": "passed", "description": "Проверка, что авторизация не осуществляется из-за некорректных данных", "parameters": [{"name": "payload", "value": "{'login': 'elenashest', 'password': '12345'}"}], "start": 1701558796151, "stop": 1701558796311, "uuid": "5faba984-3565-4bf8-86b7-84954fd9cfc5", "historyId": "9090a37d98c1aadef8eee35a367cfc5c", "testCaseId": "61afe310713b615e5766707a6449490d", "fullName": "test.test_login_courier.TestLoginCourier#test_login_courier_wrong_field_error_not_found", "labels": [{"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-QRA353I"}, {"name": "thread", "value": "13296-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_login_courier"}]}